hub/config/src/test/resources/test.json

```diff
@@ -0,0 +1 @@
tG1RD7WJ7ubQ557I0HyGY/nGlqR+SFjTh8JfDbz1ahTgznBVsesuSFgOy3Rl36bHNsNUuKLpHCLtZq/6M8G3q2aU3Z2dC1Hh7qiRgW11+Y0CI+3xTcRRxKju2QS1nSXwPLDXS+nlJ3CJs/Pfj9OG5DgvQRQ0FLzevPn8rbjIHxn"}]},{"name":"Banana Registry VSP","encryption_certificate":{"name":"/C=uk/ST=London/O=GDS/CN=gds","value":"MIIDQTCCAiigAwIBAgIBADANBgkqhkiG9w0BAQ0FADA6MQswCQYDVQQGEwJ1azEPMA0GA1UECAwGTG9uZG9uMQwwCgYDVQQKDANHRFMxDDAKBgNVBAMMA2dkczAeFw0xOTA1MTYwOTM4MjlaFw0xOTEwMjgwOTM4MjlaMDoxCzAJBgNVBAYTAnVrMQ8wDQYDVQQIDAZMb25kb24xDDAKBgNVBAoMA0dEUzEMMAoGA1UEAwwDZ2RzMIIBIzANBgkqhkiG9w0BAQEFAAOCARAAMIIBCwKCAQIAxE0gWYnXAqnQf11iWkRDDO+C9C8T+WHrpwxfTtfNILwyHnOhwZNGnO6jjGgQknfiPRVcYcLxkHS54hLlyJjqJA1EPvr/7Zb9VMibsI5wEjglq7E/iZLzsrsqAZ+98fmtodTQPk90sUbOpVi+9eK+oSylqbd4scXyWZ55xSj44xqvqVsOVLLkAFdpgTGrBd6fKx7O+i9tBS5gQVDdFytqOTrD7VrO+pofZX4LWHMoyfksPtpLdASYVnYbO4NG1dxNLq9jmFBMZXR1d8K0i0fF7D7d8mjPDFcOJZSpeLguAXoPKkLfeS6/yr/gex8jDJFtww75LOFIThQCmZMn22YQgOcCAwEAAaNQME4wHQYDVR0OBBYEFOrAl8SufPZEp51+JUisbDJpaFfHMB8GA1UdIwQYMBaAFOrAl8SufPZEp51+JUisbDJpaFfHMAwGA1UdEwQFMAMBAf8wDQYJKoZIhvcNAQENBQADggECADRaXNjOl64eqBOMxnkjD0sJwFFIAAzLqxucXcj6SikU+aS7J27fBjjqitw+dHArLry0R6QhgSGpeNEZOu31UoVzS4KL+TDxfQeK/cUgKuqnZQqkeZb3gWmZz4ynnKNUvtzmbA7bXZOgy8jQBGS/lpOprpbsleZywudW8ydn7kuvJMF9G+X7Dlc0S5Gn/PXCDYLS4JAj8uo0RLbKSqMrbnKqSoyugP7C1GPRhLAgbgwn1ozL39nAIlgbKFinuoBGb/B+ZPjpKHvkBt7p7Fngf1zEMR8RyMovKMA/kfYsmPvRxnfT13Qbd5QlNKo4sYj25FTyZfxS1teqfYwwLO4nuLXI"},"signing_certificates":[{"name":"/C=GB/ST=London/L=London/O=Cabinet Office/OU=GDS/CN=HUB Signing (20190218155358)","value":"MIIESzCCAzOgAwIBAgIQLphW6d7RItgMV/IpVCpcsjANBgkqhkiG9w0BAQsFADBOMQswCQYDVQQGEwJHQjEXMBUGA1UEChMOQ2FiaW5ldCBPZmZpY2UxDDAKBgNVBAsTA0dEUzEYMBYGA1UEAxMPSURBUCBDb3JlIENBIEcyMB4XDTE5MDIxODAwMDAwMFoXDTE5MTExNTIzNTk1OVowfTELMAkGA1UEBhMCR0IxDzANBgNVBAgTBkxvbmRvbjEPMA0GA1UEBxMGTG9uZG9uMRcwFQYDVQQKFA5DYWJpbmV0IE9mZmljZTEMMAoGA1UECxQDR0RTMSUwIwYDVQQDExxIVUIgU2lnbmluZyAoMjAxOTAyMTgxNTUzNTgpMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAp26jnUDZXaE2H3wMBtiv1Z1HHdA84dNbgqdkFrlZ5kqAYefOLwY7sTMAEd4K3xXQCkCsYOWotQieoGhhKiSolVHlGCWYq11Klqv1mRQJF7AjjRdQoHufIda0n3SiT4mcStkgaKLCaBH0gSmOkG0Bw/3wpBqPu5kvkaImPAE2eYl+z+yBMK1h+Uhka6Oo7KwiaZotkqYXbIt/zXvPXQm0RM2NY6irQzxi2JAvMY//blhIhf/iQVzDiXOcq4I5jEhkz/mZdhhyZOLD5GYPU21CAwtXfBmgCJ0Dj4ptEUtKEYULrBuxL32WP0okrWrJfPqMXCJSfVTc1usFGf7+em5/KQIDAQABo4H1MIHyMAwGA1UdEwEB/wQCMAAwVwYDVR0fBFAwTjBMoEqgSIZGaHR0cDovL29uc2l0ZWNybC50cnVzdHdpc2UuY29tL0NhYmluZXRPZmZpY2VJREFQQ29yZUNBRzIvTGF0ZXN0Q1JMLmNybDAOBgNVHQ8BAf8EBAMCB4AwHQYDVR0OBBYEFHrV0ViKwincM5p7GuVeaDGFlZzjMB8GA1UdIwQYMBaAFPj8Sbu0P8bGhufBhHon7HjiOC4dMDkGCCsGAQUFBwEBBC0wKzApBggrBgEFBQcwAYYdaHR0cDovL3N0ZC1vY3NwLnRydXN0d2lzZS5jb20wDQYJKoZIhvcNAQELBQADggEBAJPXenRSno3b2Tdtps+BZLYHyphOw40ytW0fGc9vdPrQr/t7qEnsN2RbynvbZI3gmdx+b/A7eAXaPTpWbGfZOnxnGJ4lPYWHPX4GKN67Fg0NK6/ntiaoEhqr4zzVNFI+mEXbfu4fXVO+AAqqtXYCiTQxRCAiBLTOvXI5Qe7IoZUHY+4XTfkwPe2mEIZFIurHedbrGZOxTvN3H1GXA3qW3H32sN4qUHA6xPnszcMtzXvCxWmR00nGWeIEtbc3Xd/G/uXXbzpEOGzoA3cu4RShzg3rU1k8yEbuOew0724K3AaTYv9xJDn+mc/Msft8ZAWhTPsykhn1eO8JewlHZM4XAX0="}]}]}
```

Comment:
Can we name this file to indicate what it is for, like `remote-config-test.json`?

Reply:
Moved the test file and amended the test